client/my-sites/checkout/checkout-system-decider.js

```diff
@@ -115,7 +115,7 @@ export default function CheckoutSystemDecider( {
 	if ( 'composite-checkout' === checkoutVariant ) {
 		return (
 			<CheckoutErrorBoundary
-				errorMessage={ translate( 'Sorry. there was an error loading this page' ) }
+				errorMessage={ translate( 'Sorry, there was an error loading this page.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.